modules/programs/thefuck.nix

```diff
@@ -27,15 +35,17 @@ in {
     };
   };
 
-  config = mkIf cfg.enable {
+  config = let
+    cmd = ''
+      eval "$(${cfg.package}/bin/thefuck --alias${
+        optionalString cfg.instantMode " --enable-experimental-instant-mode"
+      })"
+    '';
```

Comment:
maybe you can extract the argument into a new variable

Reply:
ok, i've moved it to its own variable.